addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -31,9 +31,10 @@
                         <Percentuale t-esc="format_numbers(abs(line.discount))"/>
                     </ScontoMaggiorazione>
                     <PrezzoTotale t-esc="format_monetary(price_subtotal, currency)"/>
-                    <AliquotaIVA t-if="line.tax_ids.amount_type == 'percent'" t-esc="format_numbers(line.tax_ids.amount)"/>
-                    <AliquotaIVA t-elif="line.tax_ids.amount_type != 'percent'" t-esc="'0.00'"/>
-                    <Natura t-if="line.tax_ids.l10n_it_has_exoneration" t-esc="line.tax_ids.l10n_it_kind_exoneration"/>
+                    <t t-set="line_taxes" t-value="line.tax_ids.filtered(lambda x: (x.amount_type == 'percent' and x.amount >= 0.0 and not x.l10n_it_pension_fund_type) or x.amount_type != 'percent')"/>
+                    <AliquotaIVA t-if="line_taxes.amount_type == 'percent'" t-esc="format_numbers(line_taxes.amount)"/>
+                    <AliquotaIVA t-elif="line_taxes.amount_type != 'percent'" t-esc="'0.00'"/>
+                    <Natura t-if="line_taxes.l10n_it_has_exoneration" t-esc="line_taxes.l10n_it_kind_exoneration"/>
```

Comment:
are you sure we can do this? since `l10n_it_pension_fund_type` is from the new module

Reply:
You're right...
We were still to decide if the new module was on auto-install, that's it.
I don't like the idea to have two different templates for maintenance reasons, it would duplicate the necessary work.